a

```diff
@@ -0,0 +1,487 @@
+[33mcommit e4195ceda64f9844b686df14ab04d22a96ff79e5[m[33m ([m[1;36mHEAD -> [m[1;32mCraftingbookFix[m[33m, [m[1;31morigin/CraftingbookFix[m[33m)[m
+Author: James Green <james@p.ckplace.org>
```

Comment:
This file should be gitignored/removed from your branch to clean up this diff at least